docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -80,11 +77,12 @@ az rg ls
 az vm ls
 ```
 
+Do not include `az` as part of the command.
+
 Aliases can also be shortcuts for complete commands. The next example lists available resource groups and their locations in table output:
 
-```
-[ls-groups]
-command = group list --query '[].{Name:name, Location:location}' --output table
+```azurecli
+az alias create --name ls-groups --command "group list --query '[].{Name:name, Location:location}' --output table"
 ```
 
 Now `ls-groups` can be run like any other CLI command.
```

Comment:
I would include a line around here that lets users know that the value passed to `--command` must not contain newlines. This makes it clearer why all of the following examples don't span multiple lines, and will prevent a common user mistake.

Reply:
Alias would attempt to convert all excessive whitespaces and next line character `\n` to single whitespaces. So technically it is valid for users to pass in commands that span multiple lines, but the actual value that we store in the configuration file would be the whitespace-truncated string. Should I still include warning?

Reply:
No, that's fine. In this case, can you put the newlines back into the examples they were removed from to make them more readable? The only reason I suggested removing them was that I wasn't sure if there was whitespace-stripping performed by the alias extension before writing to the config file.